server/src/sim/test_makeworkbook.py

```diff
@@ -46,8 +46,12 @@ def test_ref_years_range(self):
 class TestOptimaGraphTable(unittest.TestCase):
 
     def test_create_table(self):
-        table = OptimaGraphTable('Beautiful graph', [{'title':'one', 'data':[1,2,3]}, \
-            {'title':'two', 'data':[2,3,5]},{'title':'three', 'data':["a","b","c"]}])
+        sheet = [{
+            "name":"GRAPH DATA",
+            "columns":[{'title':'one', 'data':[1,2,3]}, \
+            {'title':'two', 'data':[2,3,5]},{'title':'three', 'data':["a","b","c"]}]
+        }]
+        table = OptimaGraphTable('Beautiful graph', sheet)
         path = '/tmp/test_graph.xlsx'
```

Comment:
I don't think is style. If tests are hard to read it is a reflection of the actual implementation that can be improved. Another question is if it is worth to improve the implementation and then consequently the test; I can't decide on this.

Point 3 still holds any case.